cmd/gen-release-notes/templates/minorReleaseNotes.md

```diff
@@ -10,13 +10,16 @@ This is an automatically generated rough draft of the release notes and has not
 
 # Release Notes
 
+## Deprecated
```

Comment:
Would this be better at the end of the page, instead of the first section?

Also maybe a more specific title: `Deprecated Features`?

Reply:
I was originally thinking "deprecated features", but we also deprecate APIs and other content. As far as listing it at the end, would users be more likely to miss it in that case?

Reply:
Hmm, maybe `Deprecation Notices`? Just a suggestion.

I don't think people will miss it at the end, but maybe they will. I just think that stuff that we've done wrong and are now getting rid of isn't what we want to highlight first :)

Reply:
I don't necessarily see it as stuff we did wrong. Just functionality that we decided we could do better or that no one used. Evolution :) Also, these are changes that users should be made aware of because there's a migration path now where they can explore new features in their own time but they will be breaking changes soon. 

`Deprecation Notices` sounds good. I'll add that. Thanks